MetadataExtractor/Formats/QuickTime/QuickTimeTiffHandler.cs

```diff
@@ -22,7 +24,11 @@ public override void SetTiffMarker(int marker)
             {
                 throw new TiffProcessingException($"Unexpected TIFF marker: 0x{marker:X}");
             }
-            PushDirectory(new T());
+            var directory = new T
+            {
+                Parent = Directories.OfType<TParent>().SingleOrDefault()
```

Comment:
This will throw if there's more than one instance of `TParent`, which in general happens frequently.

We need to be explicit and pass the parent to the ctor.